src/components/10-atoms/input-text/package.json

```diff
@@ -29,7 +29,6 @@
     "url": "https://github.com/axa-ch/patterns-library/issues"
   },
   "dependencies": {
-    "@axa-ch/materials": "^2.0.2",
```

Comment:
@danyball this might break the build. colors and fonts are injected via the core but not `@import '../../00-materials/styles/mixins/check-mark-animated';` which is imported in `index.scss`. Please remove all node_modules and then `npx lerna clean && npm i` and see if it works

Reply:
I remember that I mentioned this exact problem, when we discussed the "global injections" topic back then.

But the team decided, that the drawbacks of not doing that are bigger.

Reply:
Forgot the references at scss files. But it seems to work --> https://axa-ch.github.io/plib-feature/bugfix/1725_arrowicon/?path=/story/components-atoms-input-text--input-text

Reply:
I think we do not need the reference here. I could find `check-mark-animated`-css-code in dist/index.js.

Reply:
nice ok then also this is injected not just colors and  fonts